pysrc/bytewax/connectors/demo.py

```diff
@@ -92,7 +92,9 @@ def list_parts(self) -> List[str]:
         return [self._metric_name]
 
     @override
-    def build_part(self, for_part: str, resume_state: Optional[_RandomMetricState]):
+    def build_part(
+        self, _step_id: str, for_part: str, resume_state: Optional[_RandomMetricState]
```

Comment:
When overriding methods we should probably keep the original name of the kwarg, so `step_id` instead of `_step_id`, even if it's not used, so that if we pass explicitly named arguments to the subclass it still works. Same for all the other overrides in this PR